s3tests_boto3/functional/__init__.py

```diff
@@ -160,6 +160,8 @@ def setup():
         raise RuntimeError('Your config file is missing the "s3 tenant" section!')
     if not cfg.has_section("iam"):
         raise RuntimeError('Your config file is missing the "iam" section!')
+    if not cfg.has_section("webidentity"):
```

Comment:
This will cause the same issues as Eric brought up with requiring the IAM section. I would advise you to open up a different PR figuring out that issue and then use the same logic you have with that fix for requiring webidentity section only when those tests are being run.

Reply:
Yeah sure.